docs/rfcs/001-recent-commits.md

```diff
@@ -0,0 +1,87 @@
+# Recent commit view
+
+## Status
+
+Proposed
+
+## Summary
+
+Display the most recent few commits in a chronologically-ordered list beneath the mini commit editor. Show commit author and committer usernames and avatars, the commit message, and relative timestamp of each.
+
+## Motivation
+
+* Provide useful context about recent work and where you left off.
+* Allow user to easily revert and reset to recent commits.
+* Make it easy to undo most recent commit action, supersede amend check box.
+* Reinforce the visual "flow" of changes through being unstaged, staged, and now committed.
+* Provide a discoverable launch point for an eventual log feature to explore the full history.
+* Achieve greater consistency with GitHub desktop:
+
+![desktop](https://user-images.githubusercontent.com/7910250/36570484-1754fb3c-17e7-11e8-8da3-b658d404fd2c.png)
+
+## Explanation
+
+If the active repository has no commits yet, display a short panel with a background message: "Make your first commit".
+
+Otherwise, display a **recent commits** section containing a sequence of horizontal bars for each of the top three commits reachable from the current `HEAD`, with the most recently created commit on top. The user can resize the recent commits section. As it is expanded or shrunk, the number of visible commits is changed responsively.
```

Comment:
Also show commits that are on remote branch but not local (as a result of auto-fetch). Punt to show in full-featured log? Accordion style (expand/collapse). @simurai make mock-up to feel it out